fe/fe-core/src/main/java/com/starrocks/planner/RuntimeFilterDescription.java

```diff
@@ -59,6 +59,8 @@ public class RuntimeFilterDescription {
 
     private boolean onlyLocal;
 
+    private boolean topNFilter;
```

Comment:
How about use an enum instead of several boolean flags to differentiate operator types